show/main.py

```diff
@@ -1496,6 +1508,8 @@ def version(verbose):
     click.echo("HwSKU: {}".format(platform_info['hwsku']))
     click.echo("ASIC: {}".format(platform_info['asic_type']))
     click.echo("ASIC Count: {}".format(platform_info['asic_count']))
+    if asic_info:
```

Comment:
would this be disruptive for some test cases if adding output in the middle of show version cli?

Reply:
Legit question.

If I look at `tests/show_platform_test.py`, it should not change (for now) because it has a mellanox ASIC_TYPE and not a broadcom one.

My reason for placing it in the middle was this: _the top info can then be copy-pasted into bug reports without  including machine-specific serial numbers._

If you want, I can add a test there for the broadcom model with this output included.

Reply:
B.t.w. I'd rather fetch the info once at initialization from the ASIC and get it from a state(?) db during `show version`, but that's probably more invasive.

Reply:
my thought is better to keep output backward compatibility, since we are not sure if there's other customize scripts would be broken.

Reply:
Ok ok.

So, after the second paragraph and before the docker image list then?